pandas/core/indexes/base.py

```diff
@@ -144,7 +144,7 @@
     extract_array,
     sanitize_array,
 )
-from pandas.core.indexers import deprecate_ndim_indexing
+from pandas.core.indexers.utils import deprecate_ndim_indexing
```

Comment:
can you instead import the needed routines in `__init__.py` and use an `__all__` so the original imports will work e.g. `from pandas.core.indexer import .....`